openvdb/points/AttributeArray.h

```diff
@@ -463,7 +463,11 @@ struct AttributeArray::Accessor : public AttributeArray::AccessorBase
 namespace attribute_traits
 {
     template <typename T> struct TruncateTrait { };
+#ifdef OPENVDB_USE_HALF
     template <> struct TruncateTrait<float> { using Type = half; };
+#else
+    template <> struct TruncateTrait<float> { using Type = float; }; // If half is disabled, fall back to float->float.
```

Comment:
It was unclear to me if this is the correct behavior if we don't have `half`, or if there just shouldn't be a `TruncateTrait<float>` without `half`.